test/Microsoft.ML.Benchmarks/Microsoft.ML.Benchmarks.csproj

```diff
@@ -11,7 +11,7 @@
     <None Remove="BenchmarkDotNet.Artifacts\**" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.10.14" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.11.0" />
```

Comment:
Can we move this version into https://github.com/dotnet/machinelearning/blob/master/build/Dependencies.props so it isn't duplicated?

Reply:
@eerhardt done